python/metatensor-core/tests/_tests_utils.py

```diff
@@ -144,3 +143,15 @@ def tensor_zero_len_block():
             )
         ],
     )
+
+
+def can_use_mps_backend():
+    import torch
+
+    return (
+        # Github Actions M1 runners don't have a GPU accessible
```

Comment:
This is a scandal TBH that they don't allow us to use the GPU. But I understand why.

Reply:
I think this is actually because CI is running on VM and not on raw hardware, and the VM can't access the GPU in general.